src/components/Dashboard/CreateRoom/CreateRoom.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { CreateRoom } from './CreateRoom';
+
+describe('on click', () => {
+    
+    it('calls createRoom', () => {
+        const createRoom = jest.spyOn(CreateRoom.prototype, 'createRoom');
+        const wrapper = mount(<CreateRoom />);
+        wrapper.find({ 'data-cy': 'create-room' }).simulate('click');
+        expect(createRoom).toHaveBeenCalled();
+        createRoom.mockRestore();
+    });
+
+   /* TODO: moxios when createRoom is moved into actions */
```

Comment:
I think you can still write a test with moxios even if the place the request is made from changes it'll still be called